src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
 
  public void myMethod() {
   if (CoNstANT.equals("abc")) {
 
   }
+  if (CoNstANT.equals("abc")) {
+
+  }
  }
```

Comment:
**Reporter**: FINDBUGS
**Rule**: UC_USELESS_VOID_METHOD
**Severity**: INFO
**File**: se/bjurr/violations/lib/example/OtherClass.java L16
**Source**: se.bjurr.violations.lib.example.OtherClass

Useless non-empty void method


<p>Our analysis shows that this non-empty void method does not actually perform any useful work.
Please check it: probably there's a mistake in its code or its body can be fully removed.
</p>
<p>We are trying to reduce the false positives as much as possible, but in some cases this warning might be wrong.
Common false-positive cases include:</p>
<p>- The method is intended to trigger loading of some class which may have a side effect.</p>
<p>- The method is intended to implicitly throw some obscure exception.</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*